src/lib/core/tt_compression_types.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#include <stdbool.h>
+#include "trivia/config.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+enum compression_type {
+        COMPRESSION_TYPE_NONE = 0,
+#if defined(ENABLE_TUPLE_COMPRESSION)
+        COMPRESSION_TYPE_ZSTD,
+        COMPRESSION_TYPE_LZ4,
+#endif /* !defined(ENABLE_TUPLE_COMPRESSION) */
```

Comment:
It'd be great if we could add new compression types without patching the open-source part. Please consider moving these to the private part.

Reply:
fixed